docs/reference/tasking-reference.md

```diff
@@ -21,26 +21,22 @@ construct_order_parameters(
-| `acquisition_end`   | `Union[str, datetime]`                                                        | The end date of the acquisition period in the `YYYY-MM-DD` format.                                                                            |
-| `geometry`          | `Union[FeatureCollection, Feature, dict, list, GeoDataFrame, Polygon, Point]` | Geometry of the area to be captured. It can be a POI or an AOI depending on the [collection](https://docs.up42.com/data/tasking/limitations). |
-| `tags`              | `Optional[List[str]]`                                                         | A list of tags that categorize the order.                                                                                                     |
+The returned format is `dict`.
 
-<h5> Returns </h5>
+<h5> Arguments </h5>
 
-| Type   | Description       |
-| ------ | ----------------- |
-| `dict` | Order parameters. |
+| Argument            | Overview                                                                                                                                                                                                                                         |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `data_product_id`   | **str / required**<br/>The data product ID.                                                                                                                                                                                                      |
+| `name`              | **str / required**<br/>The tasking order name.                                                                                                                                                                                                   |
+| `acquisition_start` | **Union[str, datetime] / required**<br/>The start date of the acquisition period in the `YYYY-MM-DD` format.                                                                                                                                     |
+| `acquisition_end`   | **Union[str, datetime] / required**<br/>The end date of the acquisition period in the `YYYY-MM-DD` format.                                                                                                                                       |
+| `geometry`          | **Union[FeatureCollection, Feature, dict, list, GeoDataFrame, Polygon, Point] / required**<br/>The geometry of the area to be captured. It can be a POI or an AOI depending on the [collection](https://docs.up42.com/data/tasking/limitations). |
```

Comment:
might be a good idea to also give full forms of POI and AOI here.
Up to you :)

Reply:
I don't want to do it here. As a rule, we don't explain abbreviations or complicated concepts in the reference, but rather provide a link. The provided link here contains links to explanations of what POIs and AOIs are.